spec/classes/server_spec.rb

```diff
@@ -37,6 +37,9 @@
           'rsync'
         end
         it { is_expected.to contain_service(servicename) }
+        if facts[:osfamily] == 'RedHat' && Integer(facts[:operatingsystemmajrelease]) >= 8
```

Comment:
Small note:

facts[:os][:family] and facts[:os][:release][:major] would be more consistent with the existing logic in L33.